### PR TITLE
Implement js package.

### PR DESCRIPTION
**Pull request by mattcollier**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
-# json-ld-patch
-JSON-LD Patch
+# json-ld-patch-context
+JSON-LD Patch Context
```
Comment (dlongley) [inline README.md]: This repo would be for the spec too, so probably best not to make this edit.

**Review by dlongley — APPROVED** [review]
One nit. Approving.

File: contexts/json-ld-patch-v1.jsonld [hunk 2]
```diff
@@ -0,0 +1,22 @@
+{
+  "@context": {
+    "id": "@id",
+    "type": "@type",
+    "jldp": "https://w3id.org/json-ld-patch#",
+    "xsd": "http://www.w3.org/2001/XMLSchema#",
+    "add": "jldp:add",
+    "copy": "jldp:copy",
+    "frame": {"@id": "jldp:from", "@type": "@id"},
```
Comment (davidlehn) [inline contexts/json-ld-patch-v1.jsonld:9]: Should that be jldp:frame?

Comment (mattcollier) [reply contexts/json-ld-patch-v1.jsonld:9]: good catch, or should it be `from:`

Comment (mattcollier) [reply contexts/json-ld-patch-v1.jsonld:9]: created issue #2

**Pull request merged**